Mention in main CORS doc how wildcards work w/ ACAH & ACAM headers (#7999)
mdn/content: files/en-us/web/http/cors/index.md

#### Credentialed requests and wildcards

When responding to a credentialed request:

- The server **must not** specify the "`*`" wildcard for the `Access-Control-Allow-Origin` response-header value, but must instead specify an explicit origin; for example: `Access-Control-Allow-Origin: https://example.com`

- The server **must not** specify the "`*`" wildcard for the `Access-Control-Allow-Headers` response-header value, but must instead specify an explicit list of header names; for example, `Access-Control-Allow-Headers: X-PINGOTHER, Content-Type`

- The server **must not** specify the "`*`" wildcard for the `Access-Control-Allow-Methods` response-header value, but must instead specify an explicit list of method names; for example, `Access-Control-Allow-Methods: POST, GET`

> So if a request includes a `Cookie` header (the most-common type of credentials) and the response includes an `Access-Control-Allow-Origin: *` header (that is, with the wildcard), then the browser will block access to the response, and report a CORS error in the devtools console.

Reviewer: My first quick reading made me believe that it was specific to a case with `Cookie`. So I propose a small change:

```suggestion
If a request includes a credential (most commonly a `Cookie` header) and the response includes an `Access-Control-Allow-Origin: *` header (that is, with the wildcard), the browser will block access to the response, and report a CORS error in the devtools console.
```

> But if a request includes a `Cookie` header and the response includes, for example, an `Access-Control-Allow-Origin: https://example.com` header (that is, an actual origin, rather than the wildcard), then the browser will allow access to the response from the specified origin.

Reviewer: ```suggestion
But if a request includes a credential (like the `Cookie` header) and the response includes an actual origin rather than the wildcard (like, for example, `Access-Control-Allow-Origin: https://example.com`), then the browser will allow access to the response from the specified origin.
```
